packages/url_launcher/url_launcher_platform_interface/lib/url_launcher_platform_interface.dart

```diff
@@ -47,15 +47,14 @@ abstract class UrlLauncherPlatform extends PlatformInterface {
   ///
   /// For documentation on the other arguments, see the `launch` documentation
   /// in `package:url_launcher/url_launcher.dart`.
-  Future<bool> launch(
-    String url, {
-    @required bool useSafariVC,
-    @required bool useWebView,
-    @required bool enableJavaScript,
-    @required bool enableDomStorage,
-    @required bool universalLinksOnly,
-    @required Map<String, String> headers,
-  }) {
+  Future<bool> launch(String url,
+      {@required bool useSafariVC,
+      @required bool useWebView,
+      @required bool enableJavaScript,
+      @required bool enableDomStorage,
+      @required bool universalLinksOnly,
+      @required Map<String, String> headers,
+      @required String windowName}) {
```

Comment:
`windowName` must be optional, not `@required`. Also, to follow Flutter convention, this'd be better called: `webOnlyLinkTarget` (or similar, so people know at a first glance that they'll set the target attribute of a link in web only).